packages/appium/docs/en/ecosystem/index.md

```diff
@@ -13,36 +13,52 @@ learn more about the inspector here: [Appium Inspector](https://github.com/appiu
 
 ## Drivers
 
-You can't use Appium without at least one driver! Here are the drivers that are officially
-recognized by Appium. Click on the link for each driver to see the specific installation
+You can't use Appium without at least one driver! Here are the drivers that are currently
+maintained by the Appium team. Click on the link for each driver to see the specific installation
 instructions and documentation for that driver.
 
 To learn more about what drivers are and how they work, check out the [Driver
 Intro](../intro/drivers.md)
 
-|Driver|Installation Key|Platform(s)|Mode(s)|Support|
-|-|-|-|-|-|
-|[XCUITest](https://github.com/appium/appium-xcuitest-driver)|`xcuitest`|iOS|Native, Hybrid, Web|Appium Team|
-|[UiAutomator2](https://github.com/appium/appium-uiautomator2-driver)|`uiautomator2`|Android|Native, Hybrid, Web|Appium Team|
-|[Espresso](https://github.com/appium/appium-espresso-driver)|`espresso`|Android|Native|Appium Team|
-|[Mac2](https://github.com/appium/appium-mac2-driver)|`mac2`|macOS|Native|Appium Team|
-|[Safari](https://github.com/appium/appium-safari-driver)|`safari`|macOS, iOS|Web|Appium Team|
-|[Gecko](https://github.com/appium/appium-geckodriver)|`gecko`|macOS, Windows, Linux, Android|Web|Appium Team|
-|[Windows](https://github.com/appium/appium-windows-driver)|`windows`|Windows|Native|Community / Microsoft|
-|[Flutter](https://github.com/appium-userland/appium-flutter-driver)|`flutter`|iOS, Android|Native|Community|
-|[Tizen](https://github.com/Samsung/appium-tizen-driver)|`tizen`|Android|Native|Community / Samsung|
-|[Youi](https://github.com/YOU-i-Labs/appium-youiengine-driver)|`youiengine`|iOS, Android, macOS, Linux, tvOS|Native|Community / You.i|
+|Driver|Installation Key|Platform(s)|Mode(s)|
+|--|--|--|--|
+|[Espresso](https://github.com/appium/appium-espresso-driver)|`espresso`|Android|Native|
+|[Gecko](https://github.com/appium/appium-geckodriver)|`gecko`|macOS, Windows, Linux, Android|Web|
+|[Mac2](https://github.com/appium/appium-mac2-driver)|`mac2`|macOS|Native|
+|[Safari](https://github.com/appium/appium-safari-driver)|`safari`|macOS, iOS|Web|
+|[UiAutomator2](https://github.com/appium/appium-uiautomator2-driver)|`uiautomator2`|Android|Native, Hybrid, Web|
+|[XCUITest](https://github.com/appium/appium-xcuitest-driver)|`xcuitest`|iOS|Native, Hybrid, Web|
 
 And of course, you can install any other drivers you find out there by using the Appium driver CLI.
 
+### Other drivers
+
+These drivers are not maintained by the Appium team and can be used to target additional platforms.
+
+|Driver|Installation Key|Platform(s)|Mode(s)|Supported By|
+|--|--|--|--|--|
+|[Flutter](https://github.com/appium-userland/appium-flutter-driver)|`--source=npm appium-flutter-driver`|iOS, Android|Native|Community|
```

Comment:
Yeah, I think it would be fair to sort the list alphabetically

Reply:
yep i figured we should start sorting them all by alpha